src/con_duct/suite/main.py

```diff
@@ -32,7 +13,7 @@ def execute(args: argparse.Namespace) -> int:
     return result
 
 
-def main() -> None:
+def main() -> None:  # pragma: no cover
```

Comment:
why "no cover"?   can make it

```suggestion
def main(args=None) -> None:  # pragma: no cover
```

and pass `args` below into `parse_args` and thus test parsing or at least e.g. `--help` to produce desired output etc

see e.g. smth similar in https://github.com/datalad/datalad/blob/HEAD/datalad/cli/main.py#L61 and then https://github.com/datalad/datalad/blob/HEAD/datalad/cli/tests/test_main.py#L113 and there on

Reply:
I originally did this because we cant naively use the `Arguments` class like we do in main since the suite has multiple variants of potential args depending on the subparsers.

Ive added some basic tests but probably some room for improvement tbh.